src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -26,6 +26,8 @@ class DoctrineChoiceLoader implements ChoiceLoaderInterface
 {
     /**
      * @var ChoiceListFactoryInterface
+     *
+     * @deprecated since 3.1 to be removed in 4.0.
```

Comment:
In fact, this property, it's assignment and the constructor parameter can simply be removed (and then ignore any factory passed as first argument to the constructor).

It's a private property, so it couldn't be used by the outside world so there is no need for backwards compatibility.